src/index.ts

```
@@ -946,90 +918,153 @@ document.body.addEventListener('touchstart', (e) => {
 // #endregion
 
-// qs open actions
-if (!reconnectOptions) {
-  downloadAndOpenFile().then((downloadAction) => {
-    if (downloadAction) return
-    if (appQueryParams.reconnect && process.env.NODE_ENV === 'development') {
-      const lastConnect = JSON.parse(localStorage.lastConnectOptions ?? {})
+// immediate game enter actions: reconnect or URL QS
+const maybeEnterGame = () => {
+  const waitForConfigFsLoad = (fn: () => void) => {
+    let unsubscribe: () => void | undefined
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)